pkg/services/navtree/navtreeimpl/navtree.go

```diff
@@ -81,9 +81,9 @@ func (s *ServiceImpl) GetNavTree(c *contextmodel.ReqContext, prefs *pref.Prefere
 	hasAccess := ac.HasAccess(s.accessControl, c)
 	treeRoot := &navtree.NavTreeRoot{}
 
-	treeRoot.AddSection(s.getHomeNode(c, prefs))
+	//treeRoot.AddSection(s.getHomeNode(c, prefs))
```

Comment:
All changes in this file are aimed at hiding parts in the nav tree that we don't want our uses to see in the dashboards app. It's worth pointing out that by removing the elements from the `navtree` wholesale, the functionality of each of the apps is affected (or possibly completely removed). This is why for some items we don't want to see in the `navtree` we don't alter any code here, but handle this elsewhere (see `MegaMenu` changes).